.github/workflows/trunk.yml

```diff
@@ -0,0 +1,136 @@
+        uses: ./.github/workflows/_setup.yml
+        with:
+          NODE_VERSION: ${{ matrix.node-version }}
+          GITHUB_SHA: ${{ github.sha }}
+          GITHUB_REF: ${{ github.ref }}
+
+      - name: Run ${{ matrix.script }}
+        run: yarn ${{ matrix.script }}
+
+      - name: Upload coverage results to Code Climate
+        if: ${{ matrix.script == 'test:ci' }}
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /tmp/cc-test-reporter
+          chmod +x /tmp/cc-test-reporter
+          /tmp/cc-test-reporter after-build --coverage-input-type lcov --exit-code 0
+        env:
+          CC_TEST_REPORTER_ID: 3f4c9a9d57ded045e0f9ab5d23e5bbcbf709bb85637bea555f1233e72134b818 # TODO: better store it in the repository secrets. Name the variable: CC_TEST_REPORTER_ID. Then delete this line and uncomment the next one
+          # CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+
+  trunk-integration-test:
+    runs-on: ubuntu-latest
+    needs: trunk-build
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+        script:
+          - integration:ng7
+          - integration:ng8
+          - integration:ng9:ivy:off
+          - integration:ng9:ivy
+          - integration:ng10:ivy:off
+          - integration:ng10:ivy
+          - integration:ng11:ivy:off
+          - integration:ng11:ivy
+          - integration:ng12:ivy
+          - integration:ng13:ivy
+
+    steps:
+      - name: Setup
+        uses: ./.github/workflows/_setup.yml
+        with:
+          NODE_VERSION: ${{ matrix.node-version }}
+          GITHUB_SHA: ${{ github.sha }}
+          GITHUB_REF: ${{ github.ref }}
+
+      - name: Run ${{ matrix.script }}
+        run: yarn ${{ matrix.script }}
+
+  trunk-bundlesize:
+    runs-on: ubuntu-latest
+    needs: trunk-integration-test
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+        script: [bundlesize]
+
+    steps:
+      - name: Setup
+        uses: ./.github/workflows/_setup.yml
+        with:
+          NODE_VERSION: ${{ matrix.node-version }}
+          GITHUB_SHA: ${{ github.sha }}
+          GITHUB_REF: ${{ github.ref }}
+
+      - name: Run ${{ matrix.script }}
+        run: yarn ${{ matrix.script }}
+
+  trunk-publish:
+    runs-on: ubuntu-latest
+    environment: npm-publish
+    needs: [trunk-build, trunk-test, trunk-integration-test, trunk-bundlesize]
+
+    steps:
+      - name: Setup
+        uses: ./.github/workflows/_setup.yml
+        with:
+          NODE_VERSION: 16.x
+          GITHUB_SHA: ${{ github.sha }}
+          GITHUB_REF: ${{ github.ref }}
+
+      - name: Development release - publish development builds to all @ngxs packages
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+          yarn publish:dev
+          echo "Security: remove ~/.npmrc from the runner overwriting it with zeros several times."
+          find ~/.npmrc -exec shred -fuz {} +
```

Comment:
Same here, we need to copy in the build artefacts before this step

Reply:
same as above - https://github.com/ngxs/store/pull/1852/commits/016b129c3afc993f28d3f337dd193a01a8980f63